File: javascript/Day_2/Question_01.js
numbers = [10, -5, 3, -1, 0, 7, -8, 12, -3];

let count = 0;

const NumberChecker = () => {
  for (let i = 0; i < numbers.length; i++) {
    if (numbers[i] > 0) {
      count++;
    }
  }
  console.log(`total positive number is ${count}`);
};

NumberChecker();

File: dsa/Day_3/if_else/Question_01.cpp
#include<iostream>
using namespace std;

#define END endl << endl

int main(){
    int num;

    cout << "enter a number: ";

    cin >> num;

    if(num > 0){
        cout << "your number is positive numbers " << END;
    }else if(num == 0){
        cout << "number is zero" << END;
    }else{
        cout << "your number is negative numbers " << END;
    }
}